recipe/meta.yaml

```diff
@@ -193,7 +197,7 @@ outputs:
         - {{ pin_subpackage("libgomp", exact=True) }}
         - {{ pin_subpackage("openmp_impl", max_pin=None) }}
     build:
-      number: {{ build_num }}
+      string: openmp_impl_h{{ PKG_HASH }}_{{ build_num }}
```

Comment:
Same here. We will only have one `libgcc-ng`.

Reply:
See my comments above.